roles/squid/defaults/main.yml

```diff
@@ -29,14 +32,16 @@ squid_image: "{{ docker_registry_squid }}/ubuntu/squid:{{ squid_tag }}"
 squid_container_name: squid
 
 squid_allowed_addresses:
+  - aptly.airgap.services.osism.tech
+  - bandersnatch.airgap.services.osism.tech
+  - registry.airgap.services.osism.tech
+  - wormhole.airgap.services.osism.tech
   - harbor.services.osism.tech
-  - index.docker.io
-  - quay.io
-  - minio.services.osism.tech
-  - archive.ubuntu.com
-  - download.docker.com
-  - packagecloud.io
-  - github.com
-  - galaxy.ansible.com
-  - pypi.org
-  - .opendev.org
+#  - minio.services.osism.tech
+#  - archive.ubuntu.com
```

Comment:
This change is only for containers, I think we'll need to keep the other targets for a bit longer?